### PR TITLE
Fixing paths for wham scripts and unziping command for noise

### PR DESCRIPTION
**Pull request by etzinis**

### CODE REVIEW

File: egs/wham/ConvTasNet/local/prepare_data.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-wav_dir=tmp
-out_dir=tmp
+wav_dir=/mnt/data/wsj0-mix/wsj0
+out_dir=/mnt/data/wham
```
Comment (mpariente) [inline egs/wham/ConvTasNet/local/prepare_data.sh]: I don't understand these lines, sorry. 
These arguments are supposed to be parsed from `run.sh` directly, these are just default values, we can keep the `tmp` as before.

File: egs/wham/ConvTasNet/local/prepare_data.sh [hunk 2]
```diff
@@ -11,6 +11,7 @@ mkdir -p $out_dir
 echo "Download WHAM noises into $out_dir"
 # If downloading stalls for more than 20s, relaunch from previous state.
 wget -c --tries=0 --read-timeout=20 https://storage.googleapis.com/whisper-public/wham_noise.zip -P $out_dir
+unzip $out_dir/wham_noise.zip -d $out_dir
```
Comment (mpariente) [inline egs/wham/ConvTasNet/local/prepare_data.sh]: Nice catch, can we redirect it in the logs  with this `>> logs/unzip_wham.log` please?

File: egs/wham/ConvTasNet/local/prepare_data.sh [hunk 3]
```diff
@@ -20,11 +21,9 @@ mv $out_dir/wham_scripts.tar.gz $out_dir/wham_scripts
 
 wait
 
-unzip $out_dir/wham_noise.zip $out_dir >> logs/unzip_wham.log
-
 echo "Run python scripts to create the WHAM mixtures"
 # Requires : Numpy, Scipy, Pandas, and Pysoundfile
-cd $out_dir/wham_scripts
+cd $out_dir/wham_scripts/wham_scripts
```
Comment (mpariente) [inline egs/wham/ConvTasNet/local/prepare_data.sh]: My bad, I added an additional folder level in vain for the untar. Let's change the lines 18-20 instead, it makes more sense.

File: egs/wham/ConvTasNet/local/prepare_data.sh [hunk 4]
```diff
@@ -11,6 +11,7 @@ mkdir -p $out_dir
 echo "Download WHAM noises into $out_dir"
 # If downloading stalls for more than 20s, relaunch from previous state.
 wget -c --tries=0 --read-timeout=20 https://storage.googleapis.com/whisper-public/wham_noise.zip -P $out_dir
+unzip $out_dir/wham_noise.zip -d $out_dir
 
 echo "Download WHAM scripts into $out_dir"
 wget https://storage.googleapis.com/whisper-public/wham_scripts.tar.gz -P $out_dir
```
Comment (mpariente) [inline egs/wham/ConvTasNet/local/prepare_data.sh]: Moving this 
```
mkdir -p $out_dir/wham_scripts
tar -xzvf $out_dir/wham_scripts.tar.gz -C $out_dir/wham_scripts
mv $out_dir/wham_scripts.tar.gz $out_dir/wham_scripts
```
To this
```
tar -xzvf $out_dir/wham_scripts.tar.gz -C $out_dir
mv $out_dir/wham_scripts.tar.gz $out_dir/wham_scripts
```
sounds ok to you?

File: egs/wham/ConvTasNet/local/prepare_data.sh (hunk 1 above)
Comment (etzinis) [reply egs/wham/ConvTasNet/local/prepare_data.sh]: even tmp will not work for the wavdir I just puthem there for a new user to more easily understand where to look

File: egs/wham/ConvTasNet/local/prepare_data.sh (hunk 2 above)
Comment (etzinis) [reply egs/wham/ConvTasNet/local/prepare_data.sh]: Yeap no worries

File: egs/wham/ConvTasNet/local/prepare_data.sh (hunk 3 above)
Comment (etzinis) [reply egs/wham/ConvTasNet/local/prepare_data.sh]: ok

File: egs/wham/ConvTasNet/local/prepare_data.sh (hunk 4 above)
Comment (etzinis) [reply egs/wham/ConvTasNet/local/prepare_data.sh]: cool

**Comment by mpariente** [comment]
Looks great, thank you very much !

**Pull request merged**